### PR TITLE
Display discourse user info in dispute backoffice 

### PR DESCRIPTION
**Pull request by marcondag**
This is blocked by the user custom fields. This PR can be merged after being reviewed but the story will not be complete until the user data gathered during the sign up process extension @shoshber is working on is finished and those custom fields are populated. Could also use some direction from @stephenkistner re displaying the user's groups. Took a stab but it's a little bare bones, might be enough for the back-office for now, though.

screen shot 2018-04-07 at 12 44 51 pm

### CODE REVIEW

File: controllers/Admin/DisputesController.js [hunk 1]
```diff
@@ -94,10 +95,16 @@ Admin.DisputesController = Class(Admin, 'DisputesController').inherits(RestfulCo
         .catch(next);
     },
 
-    index(req, res) {
+    async index(req, res) {
       res.locals.disputes = res.locals.results;
       res.locals.statuses = DisputeStatus.statuses;
 
+      const users = await discourse.getUsers();
```
Comment (orlando) [inline controllers/Admin/DisputesController.js]: @marcondag this can potentially grow. For now, I think we are fine but let's keep an eye on performance.

File: middlewares/Router.js [hunk 2]
```diff
@@ -47,9 +47,10 @@ routeMapper.routes.forEach(route => {
     }
 
     if (!controllerMethod) {
-      throw new Error(
+      logger.warn(
```
Comment (orlando) [inline middlewares/Router.js]: @marcondag why this change?

File: controllers/Admin/DisputesController.js (hunk 1 above)
Comment (marcondag) [reply controllers/Admin/DisputesController.js]: Actually this should change. That getUsers is paginated so it won't even work with more than 50 users or something like that. We should just get the users for the disputes being displayed on the page being viewed.

Comment (orlando) [reply controllers/Admin/DisputesController.js]: Yeah, a call with specific ids will be better, not sure if we can do that

File: middlewares/Router.js (hunk 2 above)
Comment (orlando) [reply middlewares/Router.js]: 👌

File: controllers/Admin/DisputesController.js (hunk 1 above)
Comment (marcondag) [reply controllers/Admin/DisputesController.js]: We can do something like:

```javascript
await Promise.all(ids.map(discourse.getUserById))
```

But that would create 50 requests for each page. We can probably change the pagination settings to be 10 per page, or 20 might still be reasonable. I think 50 is too many though.
